src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/PolicyTests.cs

```diff
@@ -59,6 +59,13 @@ public void TestPolicyAssignmentCRUD()
             ResourcesController.NewInstance.RunPsTest(_logger, "Test-PolicyAssignmentCRUD");
         }
 
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestPolicyAssignmentIdentity()
+        {
+            ResourcesController.NewInstance.RunPsTest(_logger, "Test-PolicyAssignmentIdentity");
```

Comment:
@pilor the `.json` file corresponding to this test is not being copied to the output directory. [Here is an example of how it should be copied in the `Commands.Resources.Test.csproj` file](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Resources/Commands.Resources.Test/Commands.Resources.Test.csproj#L721-L723).

Reply:
@cormacpayne that is embarrassing.  Fixed and rebased.

Reply:
@pilor it happens quite often, no worries 😀